project-base/webpack.config.js

```diff
@@ -43,10 +44,12 @@ Encore
                 'frontend',
                 './assets/public/frontend/svg/*.svg'
             );
-            generateWebFont(
-                'admin',
-                './assets/public/admin/svg/*.svg'
-            );
+            if (sources.isMonorepo()) {
```

Comment:
it seems that this should not be here, the condition is within the function itself, or not? :thinking:

Reply:
When I want to run optimize svgs for admin really only in monorepo. I could enter the path to the framework assets directly, without getFrameworkNodeModulesDir() method :thinking:

Reply:
my point is that when I am in a project, the condition `sources.isMonorepo()` is evaluated to `false` and therefore the body of the condition is never executed ( => webfont for admin svg is never generated)

Reply:
I understand, thank you